app/src/androidTest/java/ch/epfl/sdp/user/FakeUserRepo.kt

```diff
@@ -0,0 +1,13 @@
+package ch.epfl.sdp.user
+
+import ch.epfl.sdp.db.Callback
+
+class FakeUserRepo : IUserRepo {
```

Comment:
Try to doc new classes, even if doc is missing everywhere else

Reply:
Will be done by @Dolpic in another branch after merge